datadog/resource_datadog_synthetics_test_.go

```diff
@@ -111,7 +111,37 @@ func syntheticsTestRequest() *schema.Schema {
 
 func syntheticsTestOptions() *schema.Schema {
 	return &schema.Schema{
-		Type:     schema.TypeMap,
+		Type: schema.TypeMap,
+		DiffSuppressFunc: func(key, old, new string, d *schema.ResourceData) bool {
+			if key == "options.follow_redirects" {
+				// TF nested schemas is limited to string values only
+				// follow_redirects being a boolean in Datadog json api
+				// we need a sane way to convert from boolean to string
+				// and from string to boolean
+				oldValue, err1 := strconv.ParseBool(old)
+				newValue, err2 := strconv.ParseBool(new)
+				if err1 != nil || err2 != nil {
+					return false
+				}
+				return oldValue == newValue
+			}
+			return old == new
+		},
+		ValidateFunc: func(val interface{}, key string) (warns []string, errs []error) {
+			followRedirectsRaw, ok := val.(map[string]interface{})["follow_redirects"]
+			if ok {
+				followRedirectsStr := followRedirectsRaw.(string)
```

Comment:
As we chatted about offline, this type assertion causes an issue on TF 0.11 (works fine in 0.12) Lets try to convert this value into a string if possible to avoid breaking things.